### ISSUE TITLE
Allow downloading of specific iteration of user solution

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/cli#688]
The `exercism download` command allows me to download a user's solution by specifying its UUID. It would be great if we could also specify the specific iteration to download, as sometimes I want to check against some older code. This downloaded iteration should probably then also be downloaded to a different folder.

**Comment by kytrinyx** [comment]
Yeah, I'd love this. We need the API to provide a way to do this. I'll open an issue for that.
